modules/ocf_desktop/manifests/pulse.pp

```diff
@@ -19,4 +19,11 @@
     require => Package['libasound2-plugins'],
   }
 
+  $pulse_sink = lookup({'name' => 'pulse_sink', 'default_value' => nil})
```

Comment:
I got that from here: https://www.devco.net/archives/2016/03/13/the-puppet-4-lookup-function.php
I'm just starting to figure out how all this works together, so I'm not sure.